test/knex.test.js

```diff
@@ -0,0 +1,75 @@
+const seneca = require('seneca')
+// const Shared = require('seneca-store-test')
+const Lab = require('@hapi/lab')
+const lab = (exports.lab = Lab.script())
+const { describe, it, before } = lab
+const { expect } = require('@hapi/code')
+
+const KnexStore = require('../src/knex-store')
+const DbConfig = require('./config/database/config')
+
+describe('knex-store tests', function () {
```

Comment:
we will also need to run the standard store tests
see: https://github.com/senecajs/seneca-postgres-store/blob/master/test/postgres.test.js#L28

Reply:
That was done, I will commit the change

Reply:
Ok